src/v0/destinations/posthog/data/PHPropertiesConfig.json

```diff
@@ -91,7 +91,7 @@
     "isMobile": true
   },
   {
-    "destKey": "$useragent",
+    "destKey": "$raw_user_agent",
```

Comment:
@adityapatadia can you please attatch the publicly available doc references here?